src/aui-form-builder/assets/aui-form-builder-settings-modal-core.css

```diff
@@ -38,14 +21,55 @@
     margin: 9px 0px 0px 5px;
 }
 
-.form-builder-field-settings-cancel {
-    border-width: 0;
-}
+/* Smartphones (portrait and landscape) ----------- */
```

Comment:
You don't need add this inline comment when you already have the device width information on the css (min-device-width : 320px max-device-width : 736px).